src/core/client/stream/components/App.tsx

```diff
@@ -1,28 +1,45 @@
 import * as React from "react";
 import { StatelessComponent } from "react";
 
-import { Flex } from "talk-ui/components";
+import {
+  HorizontalGutter,
+  Tab,
+  TabBar,
+  TabContent,
+  TabPane,
+} from "talk-ui/components";
+
+import { SetActiveTabMutation } from "talk-stream/mutations";
 
 import CommentsPaneContainer from "../tabs/comments/containers/CommentsPaneContainer";
+import ProfileQuery from "../tabs/profile/queries/ProfileQuery";
 import * as styles from "./App.css";
 
 export interface AppProps {
-  activeTab: "COMMENTS" | "%future added value";
+  activeTab: "COMMENTS" | "PROFILE" | "%future added value";
+  setActiveTab: SetActiveTabMutation;
+  signedIn: boolean;
 }
 
 const App: StatelessComponent<AppProps> = props => {
-  let view: React.ReactElement<any>;
-  switch (props.activeTab) {
-    case "COMMENTS":
-      view = <CommentsPaneContainer />;
-      break;
-    default:
-      throw new Error(`Unknown tab ${props.activeTab}`);
-  }
   return (
-    <Flex justifyContent="center" className={styles.root}>
-      {view}
-    </Flex>
+    <HorizontalGutter className={styles.root}>
+      <TabBar
+        activeTab={props.activeTab}
+        onTabClick={tab => props.setActiveTab({ tab })}
+      >
+        <Tab tabId="COMMENTS">Comments</Tab>
+        {props.signedIn && <Tab tabId="PROFILE">My Profile</Tab>}
+      </TabBar>
+      <TabContent activeTab={props.activeTab}>
```

Comment:
Missing gutter here between the TabBar and the TabContent according to the design.
![image](https://user-images.githubusercontent.com/14221600/45982053-81f7f980-c057-11e8-8dd6-753112b1a4c4.png)

Reply:
👍